qutebrowser/browser/history.py

```diff
@@ -162,19 +163,21 @@ class WebHistory(sql.SqlTable):
     # one url cleared
     url_cleared = pyqtSignal(QUrl)
 
-    def __init__(self, progress, parent=None):
-        super().__init__("History", ['url', 'title', 'atime', 'redirect'],
+    def __init__(self, database: sql.Database, progress: HistoryProgress,
+                 parent: QObject = None):
+        super().__init__(database, "History", ['url', 'title', 'atime', 'redirect'],
                          constraints={'url': 'NOT NULL',
                                       'title': 'NOT NULL',
                                       'atime': 'NOT NULL',
                                       'redirect': 'NOT NULL'},
                          parent=parent)
+        self.database = database
```

Comment:
This particular SqlTable needs to have its database instance public because the history completion model needs to access it.